src/status_im/ui/screens/chat/message/gap.cljs

```diff
@@ -4,12 +4,14 @@
             [re-frame.core :as re-frame]
             [status-im.i18n.i18n :as i18n]
             [status-im.utils.datetime :as datetime]
-            [status-im.ui.screens.chat.styles.input.gap :as style]))
+            [status-im.ui.screens.chat.styles.input.gap :as style]
+            [status-im.chat.models.pin-message :as models.pin-message]))
 
 (defn on-press
   [chat-id gap-ids]
   (fn []
-    (re-frame/dispatch [:chat.ui/fill-gaps chat-id gap-ids])))
+    (re-frame/dispatch [:chat.ui/fill-gaps chat-id gap-ids])
+    (re-frame/dispatch [::models.pin-message/load-pin-messages chat-id])))
```

Comment:
not sure if it's necessary here? at least i don't see reactions and messages here , cc @cammellos

Reply:
yeh we can remove this also, not needed anymore